crypto/rsa/rsa_local.h

```diff
@@ -169,39 +169,39 @@ RSA_PRIME_INFO *rsa_multip_info_new(void);
 int rsa_multip_calc_product(RSA *rsa);
 int rsa_multip_cap(int bits);
 
-int rsa_sp800_56b_validate_strength(int nbits, int strength);
-int rsa_check_pminusq_diff(BIGNUM *diff, const BIGNUM *p, const BIGNUM *q,
```

Comment:
You'll have to admit that it does look strange to have *some* RSA functions starting with `rsa_`, and some starting with `ossl_rsa_`.  However, since this is an internal set of function, for a module that is planned to go away from public view, I'll be content with an amused smile :wink:

Reply:
I wonder why this wasn't picked up by my diff with 1.1.1?  Hmmm.